var/spack/repos/builtin/packages/libceed/package.py

```diff
@@ -15,6 +15,8 @@ class Libceed(Package, CudaPackage, ROCmPackage):
     maintainers = ['jedbrown', 'v-dobrev', 'tzanio']
 
     version('develop', branch='main')
+    version('0.10', tag='v0.10.1')
```

Comment:
@jedbrown, I included only a single 0.10 - I don't think we need to separately include 0.10.0 and 0.10.1

Reply:
It's probably better to use the real vesion number instead of `0.10`:
```suggestion
    version('0.10.1', tag='v0.10.1')
```